internal/configs/version1/nginx-plus.tmpl

```diff
@@ -2,14 +2,16 @@
 worker_processes  {{.WorkerProcesses}};
 {{- if .WorkerRlimitNofile}}
 worker_rlimit_nofile {{.WorkerRlimitNofile}};{{end}}
+{{- if .WorkerRlimitCore}}
```

Comment:
why is this needed?

Reply:
No needed used for internal, will removed.